reference/pipeline/stages/index.md

```diff
@@ -388,3 +388,15 @@ Extend Spinnaker with [predefined Webhook stages](/guides/operator/custom-webhoo
 ### Custom Job
 
 Extend Spinnaker with [predefined Run Job stages](/guides/operator/custom-job-stages/).
+
+## Hiding Stages
```

Comment:
Do you think this might work better under `/guides/operator/` ?

Reply:
Makes sense; added it as a standalone page under `/guides/operator`! All comments should be addressed; this is ready for a final look whenever you have a minute 😄